process.py

```diff
@@ -82,6 +82,39 @@ def process_csv(filename: str) -> list[dict]:
 
     booklet_only_events = orientation_events
 
+    # Check for conflicts with mandatory events and invalid events
+    mandatory_events = list(
+        orientation_event
+        for orientation_event in orientation_events
+        if "mandatory" in orientation_event["tags"]
+    )
+    for event in api_response["events"]:
+        event_start = datetime.datetime.fromisoformat(event["start"])
+        event_end = datetime.datetime.fromisoformat(event["end"])
+
+        if event_end < event_start:
+            event["tags"].append("invalid")
```

Comment:
Instead of adding tags, could you instead throw an exception? 

I assume we won't want to show these events to the user, so better to just break everything.

Reply:
i feel like it it might be good to leave them in tbh, i think its more useful for as a visual flagging system (which is good for DSL since we can just send it off to them and they can make and recommend changes) and so that we can override if needed (eg. for 24 hour events), but im up for whatever

Reply:
hm yeah tags are good, but I wonder if we could do something else where they dont end up getting seen by ppl in the website or final booklet.

Could we maybe, before merge happens and once all the fuckery in the events sheet is fixed, throw for exception for invalid events (because we probably want to fix these anyway) and print a warning in the console for events that overlap with mandatory (which may happen for long running events)?

Reply:
yaurrrr that works 👀